libs/experimental/langchain_experimental/data_anonymizer/faker_presidio_mapping.py

```diff
@@ -1,15 +1,16 @@
 import string
-from typing import Callable, Dict
+from typing import Callable, Dict, Optional
 
 
-def get_pseudoanonymizer_mapping() -> Dict[str, Callable]:
+def get_pseudoanonymizer_mapping(seed: Optional[int] = None) -> Dict[str, Callable]:
     try:
         from faker import Faker
     except ImportError as e:
         raise ImportError(
             "Could not import faker, please install it with `pip install Faker`."
         ) from e
 
+    Faker.seed(seed)
```

Comment:
This modifies Faker globally, can we set the seed only on the instance we create here

Reply:
Done! :smile: